ai_presenter/ai_presenter.py

```diff
@@ -0,0 +1,123 @@
+from ai_presenter.database import Database
+import logging
+
+
+class TextAi:
+    def __init__(self, db: Database):
+        self.actors = db.actors
+        self.scenes = db.scenes
+        self.locations = db.locations
+
+    def generate(self):
+        pass
+
+    def send(self, text) -> str:
+        pass
+
+
+class TextFake(TextAi):
+    def __init__(self, db: Database):
+        super().__init__(db)
+
+    def generate(self):
+        pass
+
+    def send(self, text) -> str:
+        logging.info(f"textfake: Sending {text}")
```

Comment:
change "textfake" to match class name "TextFake"

Reply:
A possible addition in a future release is for each class to have a `self.log = logging...` that way any of the methods can use `self.log(...)` and have it set up with the prefix